### DISCUSSION TITLE
Getting started with Open Source

### DISCUSSION THREAD

**Discussion by amoghakancharla** [bregman-arie/devops-exercises#10216] [Q&A]
Hi there! I'm Amogha Kancharla, I've been working on few cloud projects on AWS. I've also recently cleared my first AWS CP certification. I'm just getting started with Open source, would really appreciate if someone can help with contributing/ pointing me towards an easy issue. Thanks :)

**Comment by Vijaykv5** [comment]
Congratulations @amoghak-ds 👏🏻, on your recent AWS Certified Professional achievement! 

As per your experience with AWS and your growing interest in open source,I Suggest you to look for issues labeled "good first issue" with tag name "AWS" to get started. Engage with the community through communication channels like Slack or Discord to seek guidance and build connections. Begin with small tasks such as improving documentation or fixing minor issues to gain familiarity with the project's codebase. That Must be good to Go.

Best of luck in your endeavors!

**Reply by Aadi-stack** [reply]
Congratulations, @amoghak-ds
